ui/imports/shared/popups/send/models/SendModalAssetsAdaptor.qml

```diff
@@ -13,7 +13,7 @@ QObject {
 
     // Controller providing information about visibility and order defined
     // by a user (token management)
-    required property ManageTokensController controller
+    required property /*ManageTokensController*/ var controller
```

Comment:
Why the type cannot be used here?

Reply:
Because of stub `WalletAsssetsStore` having this:
```qml
    readonly property var assetsController: QtObject {
        property int revision

        function filterAcceptsSymbol(symbol) {
            return true
        }
    }
```

which obviously doesn't work

Reply:
I mean, now with the "weak" declaration it works correctly